README.md

```diff
@@ -1,54 +1,106 @@
-xblock-group-project-v2
-=======================
+# xblock-group-project-v2
+
 
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
-Configuration variables
------------------------
+## Features
```

Comment:
@e-kolpakov This XBlock has a lot of cool features, you should highlight them a bit more :) As a general suggestion for this section, I'd put the keywords before `-` in each item on a separate line and make them bold. For example:

...

**Peer feedback**

Workgroup members can provide anonymous feedback to each other.

**Grading**

Activities are either peer-graded or staff-graded:

* *Staff grading* - staff members with appropriate roles are asked to provide grade for the workgroup.
* *Peer grading* - students are asked to evaluate other workgroups' work.
* *Staff-grading fallback* - if some of the students fail to provide grades to other workgroup, staff members can interfere and provide missing grades.

...